step-capstone/src/components/Sidebars/HandleClickedTimePoint.js

```diff
@@ -159,7 +157,8 @@ export const handleClickedTimePoint = (idV, startOfDisplayDate, endOfDisplayDate
 
   let timePointEnd = new Date(timePointStart);
   timePointEnd.setTime(timePointEnd.getTime() + 30 * 60000);
-  
+  console.log(timePointStart)
+  console.log(timePointEnd)
```

Comment:
Nit: Either add context to these or remove if they were meant to be temporary.

Reply:
removed, thanks